lib/dfe/analytics.rb

```diff
@@ -143,6 +143,9 @@ def self.allowlist_pii
 
     def self.hidden_pii
       Rails.application.config_for(:analytics_hidden_pii)
+    rescue StandardError
+      Rails.logger.info('Missing configuration: analytics_hidden_pii.yml. Please create this file in the config directory')
```

Comment:
Could we possibly leave out the logging ? If a project doesn't have any hidden pii, then arguably we don't need a config file.